Backpack/Button/Classes/BPKButton.m

```diff
@@ -503,6 +532,12 @@ + (UIColor *)highlightedWhite {
     return [UIColor colorWithRed:0.871 green:0.867 blue:0.878 alpha:1];
 }
 
++ (UIColor *)highlightedWhite2 {
+    // white overlayed with gray900 at 15%
+    // TODO: Add a method to blend programatically via BPKColor
+    return [UIColor colorWithRed:1 green:1 blue:1 alpha:0.75];
```

Comment:
I recently discovered [`colorWithAlphaComponent`](https://developer.apple.com/documentation/uikit/uicolor/1621922-colorwithalphacomponent?language=objc) that might be useful here:

```
return [BPKColor.gray900 colorWithAlphaComponent:0.75];

```

Reply:
Good shout - I won't change this just yes as the exact color still needs to be agreed 👍

Reply:
Done now 😃